tui/installer/screens.py

```diff
@@ -1073,7 +1075,7 @@ def ntpvalue(answers, sn):
 
     if button == 'back': return LEFT_BACKWARDS
 
-    servers = filter(lambda x: x != "", [ntp1_field.value(), ntp2_field.value(), ntp3_field.value()])
+    servers = list(filter(lambda x: x != "", [ntp1_field.value(), ntp2_field.value(), ntp3_field.value()]))
```

Comment:
This could be rewritten to use a list comprehension to match all the other changes...